app/features/conference/components/Conference.js

```diff
@@ -9,6 +9,7 @@ import { push } from 'react-router-redux';
 
 import config from '../../config';
 import { getSetting, setEmail, setName } from '../../settings';
+import i18n from '../../../i18n';
```

Comment:
please sort this alphabetically, by import path, with deeper paths going first

Reply:
Fixed